README.adoc

```diff
@@ -480,6 +480,47 @@ _ { raise_exception }.must_raise TypeError
 
 Check the http://docs.seattlerb.org/minitest/Minitest/Expectations.html[Minitest::Expectations doc] for more information about its usage.
 
+=== Hooks [[hooks]]
+
+If using a module containing `setup` or `teardown` methods, be sure to call `super` in the test class `setup` or
+`teardown`.
+
+[source,ruby]
+----
+# bad
+module MyHelper
+  def setup
+    stub_something
+  end
+end
+
+class TestMeme < Minitest::Test
+  include MyHelper
+
+  def setup
+    do_something
+  end
+end
+
+# good
+module MyHelper
+  def setup
+    stub_something
+  end
+end
+
+class TestMeme < Minitest::Test
+  include MyHelper
+
+  def setup
+    do_something
+    super
```

Comment:
IMO, as the example code, it may be better to set up the superclass first.

```ruby
def setup
  super
  do_something
end
```

Reply:
Updated. I've also added a `teardown` example as I believe that would typically be in the opposite order.